projects/next-stepper/src/lib/next-stepper/next-stepper.component.html

```diff
@@ -27,4 +27,4 @@
     <span class="step-title">{{ thirdStep }}</span>
   </section>
 </article>
-<button (click)="next()">next</button>
+<button (click)="next()">Lib next</button>
```

Comment:
Okay, I guess this button is invisible but the title "Lib next" looks really strange :)

Reply:
This button will be removed in the next PR)